site/e2e/tests/webTerminal.spec.ts

```diff
@@ -41,19 +41,18 @@ test("web terminal", async ({ context, page }) => {
   const terminal = await pagePromise;
   await terminal.waitForLoadState("domcontentloaded");
 
+  const xtermRows = await terminal.waitForSelector("div.xterm-rows", {
+    state: "visible",
+  });
+
   // Ensure that we can type in it
-  await terminal.keyboard.type("echo hello");
+  await terminal.keyboard.type("echo he${justabreak}llo");
   await terminal.keyboard.press("Enter");
 
-  const locator = terminal.locator("text=hello");
+  // Check if "echo" command was executed
+  await xtermRows.waitForSelector('div:text-matches("hello")', {
+    state: "visible",
+  });
 
-  for (let i = 0; i < 10; i++) {
```

Comment:
for reviewers: once this loop is over, and no items have been located, nothing happens even though the test should fail